src/main.rs

```diff
@@ -4,8 +4,8 @@ use horust::Horust;
 use std::path::PathBuf;
 use structopt::StructOpt;
 
-#[macro_use]
-extern crate log;
+// #[macro_use]
+// extern crate log;
```

Comment:
This was commented out to prevent the compiler from complaining that `log` is not used.
This in turn happened because I've swapped all `debug!()`s to `println!()`s as I mentioned in the PR's body.
Will fix before merge.